spec/services/pull_request/draw_reviewer_spec.rb

```diff
@@ -0,0 +1,24 @@
+describe 'PullRequest::DrawReviewer' do
+
+  context 'when there is a reviewer with smallest reviews count' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.